### PR TITLE
ADD: add icon on deploy packages

### PR DESCRIPTION
**Pull request by aperret-dcs**

### CODE REVIEW

File: install/mysql/plugin_fusioninventory-empty.sql [hunk 1]
```diff
@@ -656,6 +656,8 @@ CREATE TABLE IF NOT EXISTS `glpi_plugin_fusioninventory_deploypackages` (
   `date_mod` datetime DEFAULT NULL,
   `uuid` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
   `json` longtext DEFAULT NULL,
+  `icon` varchar(255) DEFAULT NULL,
+  `style` varchar(10) DEFAULT NULL,
```
Comment (ddurieux) [inline install/mysql/plugin_fusioninventory-empty.sql:660]: These 2 new columns must have be managed in `update.php` page to be created when update the plugin

File: inc/deploypackage.class.php [hunk 2]
```diff
@@ -505,6 +507,44 @@ function showForm($ID, $options = []) {
       echo "<td colspan='2'></td>";
       echo "</tr>";
 
+      echo "<tr>";
+      echo "<td>";
+      echo __('Icône actuelle', 'fusioninventory') . ' : ';
```
Comment (ddurieux) [inline inc/deploypackage.class.php]: In english please

File: inc/deploypackage.class.php [hunk 3]
```diff
@@ -505,6 +507,44 @@ function showForm($ID, $options = []) {
...
+         Dropdown::EMPTY_VALUE => Dropdown::EMPTY_VALUE,
+         'fas' => __('Solid', 'fusioninventory'),
+         'fab' => __('Brands', 'fusioninventory')
+      ];
+      $options = [];
+      $rand = Dropdown::showFromArray('style', $elements, $options);
+
+      echo "</td>";
+      echo "<td style='width: 20%;'>";
+      echo __('Choisir une icône pour ce service de la boutique', 'fusioninventory') . ' :';
```
Comment (ddurieux) [inline inc/deploypackage.class.php]: In english too

**Review by ddurieux — CHANGES_REQUESTED** [review]

File: install/mysql/plugin_fusioninventory-empty.sql (hunk 1 above)
Comment (aperret-dcs) [reply install/mysql/plugin_fusioninventory-empty.sql:660]: Modification faite dans le fichier install/update.php

**Review by ddurieux — APPROVED** [review]

**Pull request merged**